migrations/migration_test.go

```diff
@@ -948,148 +947,3 @@ func TestContractMigration(t *testing.T) {
 		value,
 	)
 }
-
-func TestMigrationPanics(t *testing.T) {
-
-	t.Parallel()
-
-	t.Run("composite dictionary", func(t *testing.T) {
```

Comment:
Is the behaviour same for iterating arrays as well (i.e: array of composites)? If not maybe update this test to use an array instead of an dictionary

Reply:
nevermind, types will not be loaded even for arrays, after the changes in: https://github.com/onflow/cadence/pull/3126.

Reply:
The migration of arrays was already not loading the type, as it is manually iterating using `Count()` + for-loop with `Get()` calls